### ISSUE TITLE
When start training, seems to stuck and no feedback

### ISSUE THREAD

**Issue by irisqp** [nwojke/cosine_metric_learning#32]
@nwojke hello, thanks for your work! 
I'm training this model on VeRi DataSet, and when I run train_veri.py. here are following information occured, it seems start training, but I was hoping to see more feedback information such as current iteration, current loss etc. But no feedback occured, which makes me wondering whether the program is stucked. but then use nvidia-smi to see the GPU usage, it seems working on training, than I go to the log_dir, there are some files seems to be the training checkpoint. I trained for about 10 hours, but there's no sign to stop. 
Is there anyone who knows how long it takes to train on VeRi DataSet? Or is it normal to have no feedback while training? Thanks a lot!

---------------------------------------
Run ID:  cosine-softmax
Log directory:  ./output/veri/cosine-softmax
---------------------------------------
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE3 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 0 with properties: 
name: TITAN Xp
major: 6 minor: 1 memoryClockRate (GHz) 1.582
pciBusID 0000:03:00.0
Total memory: 11.90GiB
Free memory: 11.70GiB
I tensorflow/core/common_runtime/gpu/gpu_device.cc:906] DMA: 0 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0:   Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: TITAN Xp, pci bus id: 0000:03:00.0)

**Comment by tang0704** [comment]
Have you solved this problem?  I  encounter the same problem too. I'd appreciate it if you could give me some advice

**Comment by nwojke** [comment]
Hi irisqp,

the script doesn't print any training progress to standard output. Instead, tensorboard is used for monitoring progress. Check the README.md for instructions and let me know if you find something unclear.

With default arugments, training runs indefinitelty. You can provide ``--number_of_steps=100000`` if you want the script to terminate after a fixed number of training iterations. With default model and hyperparameters training should be converged after around 100000 iterations. Below is a plot of validation accuracy on VeRi when training for (roughly) 100000 iterations.

![export_validation](https://user-images.githubusercontent.com/25549077/56945547-d6041880-6b27-11e9-8570-d06b17a4a7c8.png)

**Comment by ngingihy** [comment]
Hey @irisqp, can you mention what did you change to run training on GPU? also  @nwojke  when you say 100000 iterations, you mean the ckpt files number?

**Comment by nwojke** [comment]
@ngingihy Yes, the suffix in the ckpt files is the iterations.

**Comment by nwojke** [comment]
There is an issue open regarding training on GPU (#39). I am closing this issue due to inactivity. Re-open if the problem is not solved.

**Comment by cekcoco** [comment]
hi ,I train my data by using the code of the author,but I can not get the precision_1 in my tensorboard.could you give me some advice?
